workshop/Lab7/README.md

```diff
@@ -88,13 +117,19 @@ We will configure the IBM Cloud Operator to maange resources on your personal IB
     ibmcloud iam service-policy-create serviceid-ico --roles Manager --resource-group-name default --region us-south
```

Comment:
In testing, it turns out you still need the Administrator role added to this permission as well. This line should be, `ibmcloud iam service-policy-create serviceid-ico --roles Manager,Administrator --resource-group-name default --region us-south`